tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932207.yaml

```diff
@@ -106,3 +106,20 @@ tests:
         output:
           log:
             no_expect_ids: [932207]
+  - test_id: 7
+    desc: Referer with command injection in URL Fragment
+    stages:
+      - input:
+          dest_addr: 127.0.0.1
+          headers:
+            Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            Host: localhost
+            User-Agent: "OWASP CRS test agent"
+            Referer: "https://www.google.com/?param=something#x=foo;cat$u+/etc$u/passwd"
+          method: GET
+          port: 80
+          uri: "/get"
+          version: HTTP/1.0
+        output:
+          log:
+            match_regex: "Matched Data: .*? found within REQUEST_HEADERS:Referer: .*"
```

Comment:
Don't use wildcards here but the full string to match. With the wildcards, we won't see the value of `TX.0` and won't know whether it is what we expect it to be.

Reply:
```suggestion
            match_regex: "Matched Data: $u found within REQUEST_HEADERS:Referer: #x=foo;cat$u /etc$u/passwd"
```

Reply:
Sorry. Fixed.

Reply:
Actually... wait.

Reply:
Ok, now it's fixed.